### PR TITLE
Release Notes updates for Power

### PR DESCRIPTION
**Pull request by ktania46**
Release Notes updates 
cc @vikram-redhat @codyhoag

### CODE REVIEW

**Comment by ktania46** [comment]
@manojnkumar Please review it

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2436,6 +2434,12 @@ $ sed -i -e 's/registry-proxy.engineering.redhat.com/registry.redhat.io/g' \
     ./redhat-operator-index-manifests/imageContentSourcePolicy.yaml \
     ./redhat-operator-index-manifests/mapping.txt
 ----
++
+* For {product-title} on IBM Power Systems the following minimum requirements apply:
```
Comment (manojnkumar) [inline release_notes/ocp-4-6-release-notes.adoc:2438]: Can we change `minimum` to `recommended`?

Comment (arcprabh) [inline release_notes/ocp-4-6-release-notes.adoc:2438]: Please mention powervm because this issue is specific to powervm.

Comment (arcprabh) [reply release_notes/ocp-4-6-release-notes.adoc:2438]: For {product-title} on IBM Power Systems on powervm, the following minimum requirements apply:

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2436,6 +2434,12 @@ $ sed -i -e 's/registry-proxy.engineering.redhat.com/registry.redhat.io/g' \
     ./redhat-operator-index-manifests/imageContentSourcePolicy.yaml \
     ./redhat-operator-index-manifests/mapping.txt
 ----
++
+* For {product-title} on IBM Power System on powervm, the following recommended requirements apply:
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2438]:
```suggestion
* For {product-title} on IBM Power Systems on PowerVM, the following requirements are recommended:
```

**Review by codyhoag — COMMENTED** [review]
One suggestion; LGTM otherwise

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc]: Updated with a force push

**Comment by codyhoag** [comment]
Confirmed minor updates with @SNiemann15 and force pushed. This is now ready to merge.

**Pull request merged**